### PR TITLE
travis support github lfs

### PR DESCRIPTION
**Pull request by nplanel**

### CODE REVIEW

**Comment by nplanel** [comment]
$ git push -f -q binaries travis-builds
batch response: This repository is over its data quota. Purchase more data packs to restore access.

File: .travis.yml [hunk 1]
```diff
@@ -50,10 +56,13 @@ script:
     - git remote add binaries ${BINARIES_REPO}
     - git fetch binaries
     - git checkout -b travis-builds binaries/travis-builds
+    - git lfs track skydive-latest
```
Comment (safchain) [inline .travis.yml]: travis is not used anymore, don't know where you saw the error ?

Comment (nplanel) [reply .travis.yml]: it's on my personal account, so lfs is not enabled as I have a free account
I was used travis, as it's sometime faster than wait for our CI
.... I think we can remove .travis.yml file

**Comment by safchain** [comment]
closing it for now

**Pull request closed without merging**